docs/source/_static/css/pkdk.css

```diff
@@ -1,29 +1,44 @@
-@import 'theme.css';
-@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@200&display=swap');
+@import "theme.css";
+@import url("https://fonts.googleapis.com/css2?family=Nunito:wght@200&display=swap");
 
 body {
-    font-family: 'Nunito', sans-serif;
+  font-family: "Nunito", sans-serif;
 }
 
-.wy-nav-side {
-    background: #383838;
-}
+.wy-menu-vertical,
+.wy-nav-side,
 .wy-side-nav-search {
-    background: #A76D60;
+  width: 320px;
```

Comment:
Is it possible to make this either 1) adjustable by user by dragging; or 2) responsive to screen size?

I assume that most people are going to read RTD on non-mobile devices, but for those using mobile phones, 320px is almost the whole width of the phone.

Reply:
Removed since the non-bold font face doesn't cause the headings to clip anymore.

Also, the side nav is collapsed into a burger menu on mobile devices and simply setting the widths to 320px break the display on mobile devices.